examples/platform/alerts_common_scenarios.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
Works on my machine, and I think it features the functionality really well. Couple of things:
* Would be great if we put some of those examples in the DevNet example snippets.
* Very minor: I would print the actual commands before the result while executing(not sure if needed, would be cool I think). Example:

`  
    # Get a single alert to work with.  This could be in an iterator

    alert = alert_list.first()
    # Here's the ID of the alert.  Use this to follow along in the console
    print("alert = alert_list.first()\n")
    print("Alert id = {}".format(alert.id))
`

Reply:
examples on dev network - https://jira.carbonblack.local/browse/CBAPI-5073.
cool idea on the printing.